core/src/main/kotlin/com/malinskiy/marathon/analytics/external/influx/InfluxDbTracker.kt

```diff
@@ -0,0 +1,39 @@
+package com.malinskiy.marathon.analytics.external.influx
+
+import com.malinskiy.marathon.analytics.internal.sub.Event
+import com.malinskiy.marathon.analytics.internal.sub.TestEvent
+import com.malinskiy.marathon.analytics.internal.sub.TrackerInternal
+import com.malinskiy.marathon.execution.TestStatus
+import com.malinskiy.marathon.test.toSafeTestName
+import org.influxdb.InfluxDB
+import org.influxdb.dto.Point
+import java.util.concurrent.TimeUnit
+
+internal class InfluxDbTracker(private val influxDb: InfluxDB) : TrackerInternal {
+    override fun track(event: Event) = when {
+        event is TestEvent &&
```

Comment:
Should we wrap this type check in some generic interface?

Reply:
you mean something like a InstanceTypeChecker?